docs/control/administering/debugging.rst

```diff
@@ -207,5 +207,5 @@ For example:
 
 
 .. _`systemd's journal`: http://www.freedesktop.org/software/systemd/man/journalctl.html
-.. _`Eliot`: https://eliot.readthedocs.org
+.. _`Eliot`: https://eliot.readthedocs.org/en/0.9.0/
```

Comment:
Leave this link as it was previously. Given the context, it makes sense to point to the latest docs here, rather than a specific version.

I don't believe the "redirect with Found" warning will prevent the docs from being green, and if I'm wrong, then we need to fix that.